x/mongo/driver/description/topology.go

```diff
@@ -91,3 +93,12 @@ func (t Topology) DiffHostlist(hostlist []string) HostlistDiff {
 
 	return diff
 }
+
+// String implements the Stringer interface
+func (t Topology) String() string {
+	serversStr := ""
```

Comment:
nit: If you want the zero value of a string, you can initialize this as `var serversStr string`

Reply:
done